app/src/main/java/com/example/beerapp/BeerListRepository.java

```diff
@@ -1,8 +1,6 @@
 package com.example.beerapp;
 
-import retrofit2.Retrofit;
-
 public interface BeerListRepository {
 
-    Retrofit getNetworkData();
+    void getNetworkData();
```

Comment:
If you use RxJava, the repository should return for example `Observable` or `Single`. So here should be 
`Single<BeerList> getNetworkData()`